python/nav/ipdevpoll/dataloader.py

```diff
@@ -40,6 +40,16 @@
 from nav.ipdevpoll.db import django_debug_cleanup, run_in_thread
 import django.db
 
+
+def load_netbox(netbox_id):
```

Comment:
Exciting stuff. My only comment so far is that this strikes me as a rather strange, round-about way of writing `manage.Netbox.objects.select_related(*related).get(id=netbox_id)`